src/components/Column.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from '@emotion/styled';
+import { Container } from '@mui/material';
+
+export const Column = styled(Container)(() => ({
```

Comment:
Check out the MUI [Stack](https://mui.com/material-ui/react-stack/) component. It solves all flexbox issues, resulting in more robust and responsive CSS and less manual styling.

Reply:
I find using styled-components superfluous because you can do all the styling inline using `sx` prop. That way, everything is in one place. And if you find yourself repeating the same `sx` prop, maybe that should be a new React component?

Reply:
I personally enjoy styled components when there are a lot of lines to make the return part more readable, but i use sx if there is just one change. I have created some components for stuff which could be used several places!